ob_intern/Controller/Short Video Post/ShortVideoPostViewController.swift

```diff
@@ -74,18 +81,29 @@ extension ShortVideoPostViewController: ShortVideoPostViewModelDelegate {
 
 extension ShortVideoPostViewController: UITableViewDataSource, UITableViewDelegate{
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return caption.count
+        return viewModel?.currentList.count ?? 0
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: String(describing: ShortVideoPostTableViewCell.self)) as? ShortVideoPostTableViewCell else {
             return UITableViewCell()
         }
-        
-        cell.setData(caption: caption[indexPath.row], imageURL: mockImageUrls[indexPath.row], hashtag: hashtag[indexPath.row])
+    
+        if let currentPost = self.viewModel?.currentList.takeSafe(index: indexPath.row) {
+            cell.setData(profilePicURL: currentPost.user?.profilePic ?? "",
+                         profileName: currentPost.user?.profileName ?? "",
+                         caption: currentPost.media?.caption ?? "",
+                         postImageURL: currentPost.media?.coverImage ?? "",
+                         videoURL: currentPost.media?.video ?? "",
+                         hashtag: currentPost.hashtag,
+                         numLikes: currentPost.numberOfLikes,
```

Comment:
ฝากแก้ให้ มัน readable ที่สุดไว้ก่อนครับ อนาคตมาอ่านมันจะอ่านง่าย
```suggestion
                         numberOfLikes: currentPost.numberOfLikes,
```